yelp/style.css

```diff
@@ -0,0 +1,173 @@
+*{
+    padding: 0;
+    margin: 0;
+}
+@font-face {
+    font-family: 'yelp1';
+    src: url(assets/font/open-sans-bold.yji-1145ff0b749fff2d31bb.woff2);
+}
+@font-face {
+    font-family: 'yelp2';
+    src: url(assets/font/open-sans-regular.yji-aff9b80e384301f2ae0c.woff2);
+}
+@font-face {
+    font-family: 'yelp3';
+    src: url(assets/font/open-sans-semibold.yji-d3abae27d3eac73b40a5.woff2);
+}
+@font-face {
+    font-family: 'yelp4';
+    src: url(assets/font/poppins-bold.yji-9690ce6275ddc5645cfa.woff2);
+}
+@font-face {
+    font-family: 'yelp5';
+    src: url(assets/font/poppins-medium.yji-dc16a3592fdb61b620cc.woff2);
+}
+
+.header{
+    color: #fff;
+    width: 100%;
+    height: 500px;
+    background-size: cover;
+    background-repeat: no-repeat;
+    animation-duration: 10s;
```

Comment:
isnt't there properties using which you can map different animation properties in a single one?